### ISSUE TITLE
Make vim.org a thing

### ISSUE THREAD

**Issue by richy58729** [vim/website_next_generation#8]
I can't remember if this was an issue with the previous _vim.org_, but it definitely is an issue with the current _vim.org_, if you ask me.

The website is hosted at _www.vim.org_. Using that instead of _vim.org_ is of course a matter of taste and apparently _www.vim.org_ has been chosen. However, **at least** make it so _vim.org_ can also be resolved to an IP address, just like with _www.vim.org_, and set up a redirection from _vim.org_ to _www.vim.org_ if you're so inclined, but please, don't leave it as it is now where typing in _vim.org_ in the address bar of your web browser leaves you only with a DNS error instead of a website. That is really unhelpful.

**Comment by k-takata** [comment]
This has been requested for years:
https://github.com/vim/vim/issues/899
https://github.com/vim/vim/issues/11259
https://github.com/vim/vim/issues/13079
https://github.com/vim/vim/issues/13202

Personally, I think it it better to redirect vim.org to www.vim.org, though.

**Comment by richy58729** [comment]
Yeah, silly me, I was looking at the closed issues, but hadn't thought of looking at [vim/vim](https://github.com/vim/vim/issues/).

As mentioned in the other issues, for redirection to even work, _vim.org_ should first be able to be resolved, so either an A or a CNAME record is needed for that. In the other issues, it has been answered 'browsers just try out putting _www._ in front', and time and time again people mention that it is dependent on how a browser has been configured and even show a screenshot where it isn't working, mentioning they have a current version of the browser.
And still, issues are being closed with the answer 'this has been like so for a long time', which is a very bad reason in my opinion. If you follow that train of thought, slavery would still haven't been abolished, because that has also been a thing for a long time, so yeah, come on...

In one issue I even see the answer of Bram Moolenaar himself: 'the maintainer of the DNS records says it doesn't work'. Then, respectfully, either the maintainer of the DNS records doesn't know how to do his/her/their job or the hosting or DNS provider is being unwilling to help with setting up something that is pretty easy to set up. I don't know which is more unfortunate.
